src/pineko/kfactor.py

```diff
@@ -145,52 +127,51 @@ def scale_subgrid(extracted_subgrid, scales_array):
     return scaled_subgrid
 
 
-def compute_orders_map(m, max_as, min_al, order_exists):
-    """Compute a dictionary with all the necessary orders to compute the requested order.
+def is_already_in(to_check, list_orders):
```

Comment:
maybe we can add a "no_logs" or similar? because it is not just plain search

Reply:
```suggestion
def is_already_in_no_logs(to_check, list_orders):
```
or something like - just to reflect more the actual content

Reply:
done in 5a31fc0674d7fbc004a851f5635af0432bd0ab86